packages/dd-trace/src/config.js

```diff
@@ -1146,7 +1147,8 @@ class Config {
       DD_AGENTLESS_LOG_SUBMISSION_ENABLED,
       DD_TEST_DYNAMIC_INSTRUMENTATION_ENABLED,
       DD_TEST_MANAGEMENT_ENABLED,
-      DD_TEST_MANAGEMENT_ATTEMPT_TO_FIX_RETRIES
+      DD_TEST_MANAGEMENT_ATTEMPT_TO_FIX_RETRIES,
+      DD_TEST_UNSHALLOW_ENABLED
```

Comment:
let's make this `DD_CIVISIBILITY_GIT_UNSHALLOW_ENABLED` so as to align with Java. 

Also: let me think if we need to go through the config changes. Maybe we can check `DD_CIVISIBILITY_GIT_UNSHALLOW_ENABLED` directly in the git code, as I see no advantage in adding it to the config